io.catenax.order_status/1.0.0/OrderStatus.ttl

```diff
@@ -0,0 +1,100 @@
+@prefix : <urn:bamm:io.catenax.order_status:1.0.0#>.
+@prefix ext-planned: <urn:bamm:io.catenax.part_as_planned:1.0.0#>.
+@prefix ext-relationship: <urn:bamm:io.catenax.assembly_part_relationship:1.1.1#>.
+@prefix ext-certificate: <urn:bamm:io.catenax.esr_certificates.esr_certificate:1.0.1#>.
+
+:OrderStatus a bamm:Aspect;
+    bamm:properties (:manufacturerOrderId [
+  bamm:property :customerOrderId;
+  bamm:optional "true"^^xsd:boolean
+] :orderPositionStatuses);
+    bamm:operations ();
+    bamm:events ();
+    bamm:preferredName "Status of Order"@en;
+    bamm:description "This aspect represents the latest status of an order, that describes the business relationship between a customer and a manufacturer along the value chain. "@en.
+:manufacturerOrderId a bamm:Property;
+    bamm:preferredName "Manufacturer Order ID"@en;
+    bamm:description "This is the order number on the manufacturer side that is given by e.g., the manufacturer's enterprise resource planning system."@en;
+    bamm:characteristic :OrderIdCharacteristic;
+    bamm:exampleValue "M-Nbr-4711".
+:customerOrderId a bamm:Property;
+    bamm:preferredName "Customer Order ID"@en;
+    bamm:description "This is the order number on the customer side that can be given by e.g., the customer's enterprise resource planning system."@en;
+    bamm:characteristic :OrderIdCharacteristic;
+    bamm:exampleValue "C-Nbr-4711".
+:orderPositionStatuses a bamm:Property;
+    bamm:preferredName "Order Position Statuses"@en;
+    bamm:description "One Order consists of one or more Order Positions. Therefore the OrderStatus consists of one OrderPositionStatus per OrderPosition (one to one matching).\n"@en;
+    bamm:characteristic :OrderPositionStatusSortedSet.
+:OrderIdCharacteristic a bamm:Characteristic;
+    bamm:description "This is a proprietary order id, generated by either the manufacturer side or customer side."@en;
+    bamm:dataType xsd:string.
+:OrderPositionStatusSortedSet a bamm-c:SortedSet;
+    bamm:description "Each OrderPositionStatus must be unique. The OrderPositionStatus within the Set are sorted by the position id."@en;
+    bamm:dataType :OrderPositionStatus.
+:OrderPositionStatus a bamm:Entity;
+    bamm:properties (:positionId :totalAllocatedQuantityOnStock :quantityOrdered :customerSiteBpns [
+  bamm:property :allocatedStocks;
+  bamm:optional "true"^^xsd:boolean
+] :requiredDateOfCompletion ext-planned:manufacturerPartId);
+    bamm:preferredName "Status of Order Position"@en;
+    bamm:description "An Order Position is described by a quantity of one material (PartAsPlanned), which is requested by customer's site to be manufactured until a given time.\nThe Order Position Status includes the production status of the Order Position including already manufactured material for the Order Position."@en.
+:positionId a bamm:Property;
+    bamm:preferredName "Position ID"@en;
+    bamm:description "This is the business identifier for an Order Position."@en;
```

Comment:
is business identifier a given concept? I haven't heard about that

Reply:
The term "business identifier" shall indicate that this id is not a technical one (UUID) but one that is used to refer to the position. From that understanding the manufacturerPartId would also be a business identifier.
Let's quickly discuss later on.

Reply:
I updated the description as aligned during discussion.